pkg/azure/actuator_test.go

```diff
@@ -245,6 +246,7 @@ func TestActuator(t *testing.T) {
 				}
 
 				objects = append(objects, targetSecret)
+				objects = append(objects, testOperatorConfig(operatorv1.CloudCredentialsModeMint))
```

Comment:
Mint mode is an invalid mode for Azure; the default is manual mode.
https://github.com/openshift/installer/blob/d7138a1f324432147ad0b1f5baf43d1db5b4249a/pkg/types/defaults/installconfig.go#L86-L87